src/planner/operator/sort.rs

```diff
@@ -2,9 +2,9 @@ use crate::expression::ScalarExpression;
 
 #[derive(Debug, PartialEq, Clone)]
 pub struct SortField {
-    expr: ScalarExpression,
-    desc: bool,
-    nulls_first: bool,
```

Comment:
what does`null_first` do?

Reply:
This field is required when converting to Arrow's SortColumn